reference/api/search.mdx

```diff
@@ -1141,7 +1141,9 @@ Excludes results below the specified ranking score.
 Excluded results do not count towards `estimatedTotalHits`, `totalHits`, and facet distribution.
 
 <Warning>
-For performance reasons, if the number of documents above `rankingScoreThreshold` is higher than `limit`, Meilisearch does not evaluate the ranking score of the remaining documents. Results ranking below the threshold are not immediately removed from the set of candidates. In this case, Meilisearch may overestimate the count of `estimatedTotalHits`, `totalHits` and facet distribution.
+When using `rankingScoreThreshold`, queries paginating results with  `limit` and `offset` may exaggerate the count of `estimatedTotalHits`. For performance reasons, if the number of documents above `rankingScoreThreshold` is higher than `limit`, Meilisearch does not evaluate the ranking score of the remaining documents. Results ranking below the threshold are not immediately removed from the set of candidates.
+
+Queries using `page` and `hitsPerPage` are unnaffected by this issue and `totalHits` should always display the correct total number of search results.
```

Comment:
Actually the whole bug is fixed and the warning can be removed. We could still warn users that the route may be slower than it was before.

```suggestion
Using `rankingScoreThreshold` with `limit` and `offset` forces Meilisearch to evaluate the ranking score of all matching documents to return an accurate `totalHits`, which can impact performance.

In contrast, queries using `page` and `hitsPerPage` avoid this overhead and may benefit from significantly faster response times.
```